### ISSUE TITLE
After the code update is complete, I have encountered this problem with accessing the node 404

### ISSUE THREAD

**Issue by focusshell** [cryptiumlabs/backerei#38]
![image](https://user-images.githubusercontent.com/42060860/58749033-3da8dd00-84b3-11e9-8c98-6c8d374047ff.png)
#

**Comment by cwgoes** [comment]
Is your node storing historical data? `backerei` needs to query data from specific blocks.

**Comment by focusshell** [comment]
I did not perform cp replication when updating the node, is it related to this?

**Comment by cwgoes** [comment]
I am not sure what you mean by "cp replication", but you will probably need a full historical data archive to use `backerei` (a state snapshot alone may be insufficient).

**Comment by focusshell** [comment]
But now the data has been updated, how can I get historical data?

**Comment by focusshell** [comment]
After Athen update, the data structure change such that the size of the ledger is only 6G, not sure if the historical data still kept. I think the upgrade already prun the historical data. Do you guys have such issue after Athen upgrade? really appreciate your prompt reply.

**Comment by focusshell** [comment]
Ok, let me rephrase my question.  After Athen upgrade, I did't use CP command to back up the historical data, I assume that as such the ledger database was prun to 6G. Do I have to keep the historical data to run Backerei?  I think an important aspect of the Athen upgrade is that we can prun the ledger and save storage space, so if I have to run Backerei, now I still have to keep all the historical ledger (which is about 200G), is my understanding correct？ if this is the case, how can i recover the prun out historical data? really appreciate your prompt reply.

**Comment by cwgoes** [comment]
You need the historical data, not in its entirety but at least for the past half-dozen cycles or so.

I don't know the precise commands to restore it, I suggest asking on the Tezos developer chat.

**Comment by gaia** [comment]
@focusshell you can check what's the oldest block your node has via
`./tezos-admin-client show current checkpoint`
note the number in `Checkpoint level:`

then get the level Backerei is trying to query, in your case `BliRTVLD.....` and plug it into tzscan. the `Level` number tzscan displays for this block should be higher than your node's `Checkpoint level:`. If it is lower, backerei can't work.

**Comment by gaia** [comment]
@cwgoes would it be possible to have backerei use rpc.tezrpc.me instead of the local node?

**Comment by cwgoes** [comment]
I don't recommend using an external node, since the data returned cannot be verified, but `backerei` will support it - just change the `host` and `port` in `~/.backerei.yaml` accordingly.

You will still need a local client binary to sign transactions.

**Comment by cwgoes** [comment]
Closing since it seems like the original question has been answered.

**Comment by gaia** [comment]
In backerei.yaml I have

```
host: rpc.tezrpc.me
port: 443
```

but the request still goes at as `Secure = False`

![image](https://user-images.githubusercontent.com/87547/59161316-11cebe00-8a96-11e9-83f7-3fa2b9a08300.png)

So the response is a 400 `The plain HTTP request was sent to HTTPS port`
